include/library/objno.h

```diff
@@ -0,0 +1,9 @@
+#ifndef OBJNO_H
+#define OBJNO_H
+
+static const char KOBJECT_INVALID_STRING[] = "Invalid";
+static const char KOBJECT_FILE_STRING[] = "File";
+static const char KOBJECT_DEVICE_STRING[] = "Device";
+static const char KOBJECT_GRAPHICS_STRING[] = "Graphics";
+
```

Comment:
Move these into the implementation file.

Reply:
Moved to implementation file.